Packs/AzureNetworkSecurityGroups/Integrations/AzureNetworkSecurityGroups/AzureNetworkSecurityGroups_test.py

```diff
@@ -157,7 +162,8 @@ def test_test_module_command_with_managed_identities(mocker, requests_mock, clie
     assert client_id and qs['client_id'] == [client_id] or 'client_id' not in qs
 
 
-def test_generate_login_url(mocker):
+@pytest.mark.parametrize('authentication_type', [AUTHORIZATION_CODE, CLIENT_CREDENTIALS_FLOW])
+def test_generate_login_url(mocker, authentication_type):
```

Comment:
revert the case for CLIENT_CREDENTIALS_FLOW (the command is only relevant for auth code flow)